vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/components/iostextinput/TextInputEventEmitter.h

```diff
@@ -2,4 +2,4 @@
 // Licensed under the MIT License.
 
 // RN has some weird include directory redirections..this forwards the include to the right place
-#include <react/renderer/components/textinput/iostextinput/TextInputEventEmitter.h>
\ No newline at end of file
+#include <react/renderer/components/textinput/iostextinput/react/renderer/components/iostextinput/TextInputEventEmitter.h>
```

Comment:
Is this really what the path looks like?

Reply:
Shockingly yes! @jonthysell